joomla/joomla-cms
# [com_fields] Merge FieldsHelperInternal into FieldsHelper

Currently we have two helpers for com_fields: `FieldsHelper` and `FieldsHelperInternal`. They require eachother and one method (`canEditFieldValue`) even exists in both helpers.

### Summary of Changes
This PR proposes to delete the `FieldsHelperInternal` class and moves the two unqie methods in it to `FieldsHelper`

### Testing Instructions
* Make sure submenus for the components still work.
* Test editing values still work according to set ACL
* Test that the gallery fields plugin works as before.

### Documentation Changes Required
None
